lib/Runtime/Base/ThreadContextInfo.cpp

```diff
@@ -418,7 +418,22 @@ ThreadContextInfo::SetValidCallTargetForCFG(PVOID callTargetAddress, bool isSetV
 #ifdef _CONTROL_FLOW_GUARD
     if (IsCFGEnabled())
     {
+#ifdef _M_ARM
+        // In theory this block could be removed after ensuring that we correctly thumb-tag
+        // all callable addresses before we start passing them around; however, currently a
+        // few places don't, and expect the thumb bit to be set later.
```

Comment:
>few places don't [](start = 11, length = 16)

Do you know of places where we don't?

It will be good to fix atleast the places you have repro for.

Reply:
My repro cases no longer repro, so I'm just going to drop that block until we find a case where it does.

Reply:
Lets run Exprgen using arm32 binaries and see how many issues we hit.

---
In reply to: [160273815](https://github.com/Microsoft/ChakraCore/pull/4514#discussion_r160273815) [](ancestors = 160273815)

Reply:
I cleaned this up a few months ago so that we are more consistent

Reply:
Exprgen found a few cases where we were passing in non-tagged addresses, which I'll add some commits to fix.